<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Reginas | For Employees</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color: black;
border-radius: 10px;

</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>24</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #fff</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;For Employees&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>241</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>36</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Reginas</string>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="buddy">
     <cstring>label</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditHowMuch">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color: black;
border-radius: 10px;

</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonTake">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>290</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 49, 113);
color: white;
border-radius: 10px;

</string>
    </property>
    <property name="text">
     <string>Take the tests</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditProblem">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>371</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
color: black;
border-radius: 10px;

</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Brief description of the fault</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSend">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>540</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 49, 113);
color: white;
border-radius: 10px;

</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>350</y>
      <width>600</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 49, 113);
border-radius: 10px;

</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
